src/modules/secondbar.js

```diff
@@ -13,6 +18,26 @@ const secondBar = () => {
   mainContainer.appendChild(form);
   form.appendChild(inputText);
   form.appendChild(addBtn);
+  let lastIndex;
+  let lastDato;
+  addBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    if (inputText.value) {
+      if (toDoTasks.length === 0) {
+        lastIndex = 0;
+      } else {
```

Comment:
- [ ] An error page shows when if a user attempts to add a task by pressing the "Enter" key on the keyboard as shown below 👇 
![image](https://user-images.githubusercontent.com/46839548/177896077-68feb2b1-a840-4d10-b9af-e4b95b3892fa.png)
In order to fix this issue, you can add a `keydown` event listener and check if the `code === 'Enter'`